### ISSUE TITLE
Reducing Tree's json editor ability 

### ISSUE THREAD

**Issue by syedjavedahmed** [josdejong/jsoneditor#1511]
Hi I am using 3 modes while loading my data into json on my webpage,

I wanted to reduce tree mode's ability.... 

I actually wanted a form like mode with insert buttons on left.

Basically, I wanted to reduce a user's ability into changing key side of string, and only allow it to either add more key value or edit the existing value side of key. 
I tried adding css in form mode, but it seems like it doesnt support it... 

Any other way of achieving it? or making changes to tree mode to make key side non-editable?

**Comment by josdejong** [comment]
Have you tried the `form` mode?

**Comment by syedjavedahmed** [comment]
Yes, Form we are using to provide to user to edit value side json only. 

For Admin we need to limit his ability from Changing Key side of a json object but to be also be able to add a new key value pair within the template!

**Comment by josdejong** [comment]
One option could be to add a button in the main menu using the option `onCreateMenu`. This button could get the selected path, and then insert a new key in the selected object.

**Comment by josdejong** [comment]
yes, so the `form` mode would disable editing keys, and the custom menu button would allow adding a new key:value pair?
